dashboard/app/models/level_group.rb

```diff
@@ -87,4 +88,135 @@ def pages
   def plc_evaluation?
     levels.map(&:class).uniq == [EvaluationMulti]
   end
+
+  # Surveys: How many students must complete a survey before any results are shown.
+  SURVEY_REQUIRED_SUBMISSION_COUNT = 5
+
+  # Surveys: Given a sublevel, and the known response to it, return a result object.
```

Comment:
We chatted about this in person and mostly resolved this, but adding a note here as well:

Since all levels can have sublevels, I think it may make sense to put this on the `Level` model to make it accessible to all levels.  We can move it once other levels need it, but it's probably simpler to put it on `Level` to begin with.